directories/src/redux/services/broker-node.js

```diff
@@ -39,22 +34,17 @@ const requestGenesisHashPoW = (currentList) =>
 const completeGenesisHashPoW = (txid, trytes) =>
   axios({
     method: "PUT",
-    url: `${API_ROOT_URL}/${API_VERSION}/demand/transactions/genesis_hashes/${txid}`,
+    url: `${API_ROOT_URL}/${API_VERSION}/demand/transactions/genesis_hashes/${
+      txid
+    }`,
     data: { trytes }
   });
 
-const requestOldGenesisHashPoW = (currentList) =>
+const claimTreasure = (seed, genesisHash, numberOfChunks) =>
```

Comment:
I dunno if this was something you changed or not, but I think we'd need to also tell the broker what sector we did.  Otherwise it won't know what PoW to do the verification for.

Reply:
just moved the function down the file, didn't change it